qiskit/visualization/pulse_v2/layouts.py

```diff
@@ -0,0 +1,189 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+r"""
+A collection of functions that decide layout of figure.
+
+Currently this module provides functions to arrange the order of channels.
+
+An end-user can write arbitrary functions with the following function signature:
+
+    ```python
+    def my_channel_layout(channels: List[Channel]) -> List[Channel]:
+        ordered_channels = []
+        # arrange order of channels
+
+        return ordered_channels
+    ```
+
+The user-defined arrangement process can be assigned to the layout of the stylesheet:
+
+    ```python
+    my_custom_style = {
+        'layout': {'channel': my_channel_layout}
+    }
+    ```
+
+The user can set the custom stylesheet to the drawer interface.
```

Comment:
```suggestion
A collection of functions that decide the layout of a figure.

Currently this module provides functions to arrange the order of channels.

An end-user can write their own layouts with by providing a function with the signature:

    ```python
    def my_channel_layout(channels: List[Channel]) -> List[Channel]:
        ordered_channels = []
        # arrange order of channels

        return ordered_channels
    ```

The user-defined arrangement process can be assigned to the layout of the stylesheet:

    ```python
    my_custom_style = {
        'layout': {'channel': my_channel_layout}
    }
    ```

The user can set the custom stylesheet to the drawer interface.
```